src/storage/store.js

```diff
@@ -19,6 +19,7 @@ export const SCHEMA = {
       properties: {
         has_agreed_to_terms: { type: "boolean" },
         has_changed_name: { type: "boolean" },
+        has_found_freeze: { type: "boolean" },
```

Comment:
small thing, but I'd probably move these boolean flags outside of `profile` -- the `profile` object was intended to be the user's profile like name and avatar. the terms/name ones we might be forced to leave for backwards compatibility.